packages/suite-desktop/e2e/support/common.ts

```diff
@@ -59,6 +61,11 @@ export const patchBinaries = async () => {
         coinjoinMiddlewarePathFrom,
         `${coinjoinMiddlewarePathTo}/WalletWasabi.WabiSabiClientLibrary`,
     );
+    // todo: for some reason, wabisabiclient lib needs to update permissions
+    fs.chmodSync(
+        `${coinjoinMiddlewarePathTo}/WalletWasabi.WabiSabiClientLibrary`,
+        fs.constants.S_IXOTH,
+    );
```

Comment:
I think it would be nice to follow the style in this file and use async/await instead of sync blocking method.

Like:
```
const chmod = promisify (fs.chmod);

...

chmod(`${coinjoinMiddlewarePathTo}/WalletWasabi.WabiSabiClientLibrary`,
        fs.constants.S_IXOTH,
);

```

The same way that is done with `mkdir`, `fileExists` and `copyFile`.